migrations/006.do.daily-node-metrics.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE daily_node_metrics (
+    metric_date DATE NOT NULL,
```

Comment:
In the other daily table (`daily_participants`), we call the date column `day`.

It makes me wonder why you chose `metric_date` as the name. Did you find the `day` name not clear enough? I don't mind using `metric_date` if there is a good argument. (Although I find it a bit long.)

It would be great to stay consistent, and if we agree to use `metric_date` instead of `day`, then we should look into renaming the existing `day` columns. (That's out of the scope of this pull request, though. It will also require synchronised changes in spark-evaluate and spark-stats.)

```suggestion
    day DATE NOT NULL,
```

Reply:
"Day" is often used in Postgres syntax. Although it's not a restricted keyword, typically it's suggested to avoid those as well to prevent confusion.

I don't have a hard opinion on it though, if you'd prefer "day"!